src/ServiceControl.Contracts/ServiceControl.Contracts.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net40;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
```

Comment:
Should we also target .NET 5?

Reply:
There's also a taskforce that upgrade projects to support .NET 6. I wonder if they are adding net6 target explicitly as well?